docs/README.md

```diff
@@ -12,6 +12,6 @@
   - [Available widgets](building-a-form/available-widgets.md)
   - [Common definitions](building-a-form/common-definitions.md)
   - [Common patterns for building forms](building-a-form/common-patterns-for-building-forms.md)
-  - [Creating custom fields and widgets](building-a-form/creating-custom-fields-and-widgets.md)
-- [How the library works](how-the-library-works/README.md)
```

Comment:
One thought about removing this topic on "How the library works": I was imagining other topics could go in here in the future. For example, in order to address [this ticket](https://github.com/usds/us-forms-system/issues/171), I was going to add a topic on how USFS uses Redux, and I assumed it would fit into a "How the library works" category, but curious on your thoughts here.

Reply:
I can help think through where to put that information. We really need these three different maps (sections) for these distinct audiences:

- People just introducing themselves to the library (Getting started)
- People who have everything set up and are building basic forms with no customizations (Building a form)
- People who are not only building forms, but building customizations on top of the library (Customizing the library)

These are what equate to "Beginner, Intermediate, and Advanced." Users do not self-identify into those groups accurately, so you have to be very explicit about the use cases.

Reply:
I guess to me "Customizing the library" feels like a topic within a larger category of advanced things someone might need to know about potentially. For example, I can foresee future topics within an advanced category related to adding features for prefill and save in progress or how the state model works.

Reply:
> I can foresee future topics within an advanced category related to adding features for prefill and save in progress or how the state model works.

Definitely. We're eventually going to want to break up "Building a form" more, depending on future improvements to the library.

Reply:
Okay that makes sense! Sounds good :)